### PR TITLE
Support Play/Pause, FF/Rewind of video using controller buttons

### PR DESCRIPTION
**Pull request by nkronlage**
This adds some simple controls to the equirect-video sample to allow play/pause and fast forward/rewind of the video.

### CODE REVIEW

**Comment by cabanier** [comment]
This looks great!

File: equirect-video.html [hunk 1]
```diff
@@ -172,6 +172,13 @@
     video.preload = 'auto';
     video.autoload = true;
 
+    const lastFrameButtonsByGamepad = new WeakMap();
+    function buttonPressedThisFrame(gamepad, lastFrameButtons, index) {
+      return (gamepad.buttons.length >= index &&
```
Comment (Artyom17) [inline equirect-video.html:177]: if index == length then ... caboom!

Comment (cabanier) [reply equirect-video.html:177]: @nkronlage do you have time to fix this?

Comment (nkronlage) [reply equirect-video.html:177]: Oops, looks like #6 fixes this. Thanks @Artyom17 !

**Pull request merged**